dask_planner/src/expression.rs

```diff
@@ -146,14 +160,36 @@ impl PyExpr {
                         Ok(fq_name) => {
                             let mut idx: usize = 0;
                             for schema in input_plans[0].all_schemas() {
-                                match schema.index_of_column(&Column::from_qualified_name(&fq_name))
-                                {
-                                    Ok(e) => {
-                                        idx = e;
-                                        break;
+                                let (schema_name_opt, column_name) =
+                                    self._schema_column_from_fqn(&fq_name);
+                                match &schema_name_opt {
+                                    Some(e) => {
+                                        match schema
+                                            .index_of_column_by_name(Some(e.as_str()), column_name)
+                                        {
+                                            Ok(e) => {
+                                                idx = e;
+                                                break;
+                                            }
+                                            Err(_e) => {
+                                                println!(
+                                                    "Schema_name_opt: {:?} - Column_Name: {:?}",
+                                                    schema_name_opt, column_name
+                                                );
+                                                panic!("Didn't find in index ...")
```

Comment:
Since this is accessible from Python, should we return a Python error here instead of panicking?